projects/swimlane/ngx-ui/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -79,6 +80,8 @@ export class DropdownComponent implements AfterContentInit, OnDestroy {
     this._closeOnMouseLeave = coerceBooleanProperty(val);
   }
 
+  @Input() @CoerceBooleanProperty() forceDownwardOpening = true;
```

Comment:
Did you mean `false` here?

Reply:
yes... my idea is to set to true when #690 is fixed.